operator/Dockerfile

```diff
@@ -29,6 +31,8 @@ FROM gcr.io/distroless/static:nonroot
 WORKDIR /
 COPY istio-chart/ istio-chart/
 COPY --from=builder /workspace/manager .
+COPY --from=builder /workspace/manifests/ .
```

Comment:
Maybe we should make this directory identical as for running locally? For now `make run` doesn't work
```suggestion
COPY --from=builder /workspace/manifests/ manifests
```

Reply:
True, I will fix this